.gitignore

```diff
@@ -69,3 +69,4 @@ playwright-e2e/.env
 # Ignore .env file
 .env
 
+playwright-e2e/tests/accessibility.spec.ts
```

Comment:
is there a reason to not version accessibility spec test class?

Reply:
removed.